scopesim_templates/metis/laser.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""WCU laser spectrum for METIS."""
+
+import numpy as np
+from astropy import units as u
+from astropy.io import fits
+
+from spextra import Spextrum
+
+from ..rc import Source, im_plane_utils
+
+
+def laser_spectrum(img_size=(12, 12),
+                   waves=np.arange(3, 6, .001)*u.um,
+                   centers=[3.39, 4.73, 5.263]*u.um,
```

Comment:
So, I'm going to make things a bit more complicated w.r.t. to the headers. (About names: `SEQ.WCU.XYZ` means header `HIERARCH ESO SEQ WCU XYZ`.) See template manual for more details.

The raw data will have several headers to indicate which laser source is on, e.g. `SEQ.WCU.LASER4` to `SEQ.WCU.LASER7` are for the 4 N-band lasers, and 1 to 3 are for the LM-band laser. Of particular interest is `SEQ.WCU.LASER2.WAVE` is the wavelength of the tunable laser.

Now I'm wondering how we could easily propagate this information from the source to the header... But I think we should park that discussion for now.